outgoingra/src/main/java/no/bankid/outgoing/ra/NotifyUserOfActivationRequestBody.java

```diff
@@ -0,0 +1,42 @@
+package no.bankid.outgoing.ra;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.Map;
+import java.util.UUID;
+
+@Schema(description = "Request body content for asking an RA to send an activation code to an endUser")
```

Comment:
Mismatch between class name and docs
```suggestion
@Schema(description = "Request body content for asking an RA to send a notification to an endUser")
```